plugin/__init__.py

```diff
@@ -1373,6 +1373,13 @@ def cardReviews(self, deck, startID):
         )
 
 
+    @util.api()
+    def getReviewsOfCard(self, card):
+        return self.database().all(
+            'select id, cid, usn, ease, ivl, lastIvl, factor, time, type from revlog where cid = ?', card
```

Comment:
Can you change this to be a dictionary instead of a simple list? Using arbitrary indices is error prone.

Reply:
Sounds good! As an additional note, I changed the API call to accept multiple cards in its argument, since the dictionary format is a lot safer to use, and there seems to be enough of a precedent in the API to read values from multiple cards at once.